src/js/script.js

```diff
@@ -76,7 +77,7 @@ export async function fetchNetworkData (networkName) {
  * @returns {string} HTML string for the network item
  */
 export function createNetworkItemHTML (network) {
-  return `
+  return htmlEscape`
```

Comment:
This is why you use React - by definition it's impossible to insert HTML from server responses